### PR TITLE
Create ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61177](https://cumulocity.atlassian.net/browse/MTM-61177)
Original PR: [7640](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7640)

[MTM-61177]: https://cumulocity.atlassian.net/browse/MTM-61177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Provide option to enable/disable Dashboard Manager in the App configuration (#7567) [GRAFT][release/cd] (#7640)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md]: The Claude AI suggested to use the following title:
```suggestion
title: Option to enable or disable Dashboard Manager in App configuration
```

File: content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61177
+version: 1021.8.0
+---
+Provide option to enable/disable Dashboard Manager in the App configuration (#7567) [GRAFT][release/cd] (#7640)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md]: The Claude AI suggested to use the following body:
```suggestion
To provide more flexibility for users, an option has been added to the App configuration which allows enabling or disabling the Dashboard Manager. Previously, the Dashboard Manager was always enabled without the possibility to turn it off. With this change, users can now decide whether they want to use the Dashboard Manager functionality in their applications or not, depending on their specific requirements. This configuration option impacts all applications where the Dashboard Manager was previously available and used.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md (hunk 1 above)
Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md]:
```suggestion
title: Option to enable or disable Dashboard Manager is added to Cockpit Application configuration
```

File: content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md (hunk 2 above)
Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md]: User is now able to disable or enable Dashboard manager from Application configuration in Cockpit Configuration. By default, Dashboard manager is enabled.

Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md]:
```suggestion
User is now able to disable or enable Dashboard manager from Application configuration in Cockpit Configuration. By default, Dashboard manager is enabled.
```

File: content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61177
+version: 1021.8.0
+---
+User is now able to disable or enable Dashboard manager from Application configuration in Cockpit Configuration. By default, Dashboard manager is enabled.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md]:
```suggestion
In the Cockpit application, it is now possible to  disable or enable the dashboard manager in the **Application configuration** page. By default, the dashboard manager is enabled.
```

File: content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Option to enable or disable Dashboard Manager is added to Cockpit Application configuration
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md]:
```suggestion
title: Option added to enable or disable the dashboard manager in the application configuration
```

**Review by BeateRixen — APPROVED** [review]

**Review by rahulc8y — APPROVED** [review]

**Pull request merged**